doc/modules/preprocessing.rst

```diff
@@ -461,38 +461,43 @@ not desired (i.e. the set of browsers was ordered arbitrarily).
 
 One possibility to convert categorical features to features that can be used
 with scikit-learn estimators is to use a one-of-K or one-hot encoding, which is
-implemented in :class:`OneHotEncoder`.  This estimator transforms each
+implemented in :class:`CategoricalEncoder`.  This estimator transforms each
```

Comment:
Given the availability of ordinal encoding from CategoricalEncoder, I think it should be described differently here.